templates/partials/character-header.html

```diff
@@ -1,18 +1,24 @@
-<div class="left-sidebar">
-    <div class="stat-title">
+<div class="char-left-sidebar">
+    <button></button>
+    <div class="char-stat-title">
         <label>Current/Max</label>
     </div>
     {{#each data.stats as |details stat|}}
-    <div class="stat-display" data-stat="{{stat}}">
+    <div class="stat-display char-stat-display" data-stat="{{stat}}">
         <label class="stat-label"><a class="stat-roll">{{localize details.label}}</a></label> 
+<<<<<<< Updated upstream
         <input class="stat" name="data.stats.{{stat}}.current" type="text" value="{{details.current}}" placeholder="0" data-dtype="Number"/>
         <input class="stat" name="data.stats.{{stat}}.max" type="text" value="{{details.max}}" placeholder="0" data-dtype="Number"/>
+=======
+        <input class="header-stat" name="data.stats.{{stat}}.current" type="number" value="{{details.current}}" placeholder="0" data-dtype="Number"/>
```

Comment:
There is still changes stashed, you will need to adress that before merging

Reply:
Something got confusing when updated to the dev instead of main. Should not be a problem in the future :P

Reply:
Fixed